src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -13431,6 +13417,18 @@ public String setLust(float lust) {
 				+ "</p>";
 		}
 	}
+
+	public void setLustNoText(float lust) {
+		if (lust < 0) {
+			setAttribute(Attribute.LUST, 0, false);
+		} else if (lust > 100) {
+			setAttribute(Attribute.LUST, 100, false);
+		} else {
+			setAttribute(Attribute.LUST, lust, false);
+		}
```

Comment:
This *might* be shortened with:
```java
		setAttribute(Attribute.LUST, Math.max(0, Math.min(lust, 100)), false);
```

Reply:
You're right, all I did was move it around and so I didn't really have a good look, but less code is better